sileo-featured.json

```diff
@@ -0,0 +1,20 @@
+{
+  "class":"FeaturedBannersView",
+  "itemSize":"{263, 148}",
+  "itemCornerRadius":10,
+  "banners":
+   [
+     {
+       "url":"https://raw.githubusercontent.com/mac-user669/cydia/master/assets/sileobanners/LotusDarkExtra.png",
```

Comment:
Could you edit the URLs to refer to my repo `/favna/cydia` instead of yours?

Reply:
*Facepalm* Sorry, hang on

Reply:
Done